### DISCUSSION TITLE
automatic install of comfy ui

### DISCUSSION THREAD

**Discussion by vpk1950** [comfyanonymous/ComfyUI#6000] [Q&A] [answered]
I am a newbie. Can anyone help in an automatrf install.bat for me load comfy ui on my windows laptop.

**Comment by LukeG89** [comment, marked as answer]
You can simply download and extract the [latest portable version](https://github.com/comfyanonymous/ComfyUI/releases/latest/download/ComfyUI_windows_portable_nvidia.7z) and use `run_nvidia_gpu.bat` or `run_cpu.bat` to start ComfyUI.

To install [ComfyUI-Manager](https://github.com/ltdrdata/ComfyUI-Manager), you need to follow [these instructions](https://github.com/ltdrdata/ComfyUI-Manager?tab=readme-ov-file#installationmethod2-installation-for-portable-comfyui-version-comfyui-manager-only)

**Reply by vpk1950** [reply]
Thank you so much.

**Reply by vpk1950** [reply]
Thank you so much.
